issue_update_prompter.py

```diff
@@ -0,0 +1,183 @@
+import dominate
+import itertools
+import copy
+import bunch
+import yaml
+from datetime import datetime
+from github import Github
+import argparse
+from dominate.tags import table, tbody, h3
+import report
+import smtplib
+from email.mime.text import MIMEText
+
+
+def make_email_body(issues):
+    title = 'BETA:\nPlease update or close the following issues so Bianca and Spencer know what is going on.'
+    d = dominate.document(title=title)
+    with d.body:
+        h3(title)
+        with table(border='1', width='1024', cellpadding='10').add(tbody()):
+            report.make_table_header()
+            decorated_issues = report.build_decorated_issues(issues)
+            decorated_issues.sort(key=lambda issue: issue.priority)
+            for decorated_issue in decorated_issues:
+                report.make_table_row(decorated_issue)
+    return str(d)
+
+
+def get_notification_from_repos(criteria, repos):
+    notification_lists = [get_notification_from_issues(criteria, repo.get_issues()) for repo in repos]
+    notifications = list(itertools.chain.from_iterable(notification_lists))
+    return notifications
+
+
+def time_to_notify(timeframe, last_updated):
+    """I'm making this a seperate function, as I think we will eventually find some 
+    odd edge cases that we will want to build into this logic (last updated 23.5 hours ago, etc). 
+    target_days (int or float): number of days at which we want to notify
+    last_updated (datetime): datetime object from github of last time issues was updated """
+    day_delta = (datetime.now() - last_updated).days
+    if 'catch-up' in timeframe and timeframe.catch-up:
+        return day_delta >= timeframe.days
+    else:
+        return day_delta == timeframe.days
+
+
+def generate_recipients(timeframe, issue):
+    recipients = copy.deepcopy(timeframe.recipients)
+    if 'AUTHORS' in recipients:
+        def debug_assignees(assignee):
+            if assignee.email is None:
+                print('''Github user {} can't be contacted as they don't 
+                      have a public email set. Found in issue {}.'''.format(assignee, issue))
+
+        list(map(debug_assignees, issue.assignees))
+        assignee_emails = [assignee.email for assignee in issue.assignees if assignee.email]
+        if not assignee_emails:
+            pass
+            # todo: what do we want to do with issues that are not assigned? PM or nothing?
```

Comment:
I'd mark the _triage_ label so that someone can just do a query on issues with the triage label and actually do the triage.

Reply:
Sounds like a good short-term fix. In the long run we might want to have an entry for PM email and send these and the missing user email addresses to them.